app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadListFragment.kt

```diff
@@ -47,22 +49,21 @@ class ThreadListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
     private val mainViewModel: MainViewModel by viewModels()
     private val threadListViewModel: ThreadListViewModel by viewModels()
 
-    private lateinit var binding: FragmentThreadListBinding
+    private val binding by lazy { FragmentThreadListBinding.inflate(layoutInflater) }
```

Comment:
Don't do that, init your fragment's binding in `onCreateView`, to be able to benefit from the `container`, because this constructor always set it to null, and for the moment I don't know if it can create problems, but we will avoid it for now